packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -1,14 +1,9 @@
 <template>
-    <a
+    <component
+        :is="wrapperComponent"
         :aria-label="linkAltText"
```

Comment:
This can be removed as we should now be getting this from `linkProperties` if we need it.